.travis.yml

```diff
@@ -1,20 +1,19 @@
-language: node_js
-node_js:
-  - "6"
-
-sudo: false
+language: generic
+dist: trusty
```

Comment:
From the docs, it sounds like turning on the `docker` under services is the prefered method https://docs.travis-ci.com/user/docker/